### PR TITLE
FIX : broken notebook + travis

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

**Comment by agramfort** [comment]
I am at scipy if you want to chat.

A few comments:
- it would be great if you guys could follow some scikit-learn linear_model conventions for namings. Like N -> n_samples, p -> n_features, lambduh -> alpha or reg or C

I also get flake8 errors:

dask_glm/algorithms.py:378:5: F841 local variable 'armijoMult' is assigned to but never used
dask_glm/algorithms.py:403:13: F841 local variable 'step' is assigned to but never used

shall I fix them in this PR?

**Comment by hussainsultan** [comment]
Thank you for the PR. I am traveling right now and will review once i get home. Sorry about the delay.

**Comment by hussainsultan** [comment]
This looks good! Thank you.  I am unsure around why Travis is failing. I will look into it some more as i find time. 

I like the suggestion for standardizing variables with sklearn. @moody-marlin any thoughts?

**Comment by agramfort** [comment]
yes travis is not happy. I now try to test the notebooks but it stalls and one test fail on the other build.

**Comment by hussainsultan** [comment]
@agramfort  i ran your branch locally and it seemed to pass without issues for python 3.5 and 2.7.  Restarting...

**Comment by agramfort** [comment]
ok let me know if travis needs more from me

**Comment by TomAugspurger** [comment]
We can do a couple things to speed up the `basic_api.ipynb` notebook.

- change `ddf.random_split` to something like 10:90
- repartition `X_train` and `y_train` to have 2 partitions

With those changes, `lm.fit` takes about 7 seconds on my computer. We'll want to note in the text that these are just for demonstration purposes. If you were fitting on the entire dataset on a cluster, having just 2 partitions likely wouldn't fit in memory.

@agramfort I can push those updates if you want. I'm curious to see if Travis finishes first though.

**Comment by TomAugspurger** [comment]
Oh, just noticed one change from the prior version. The `for i in *.ipynb...` seems to be picking up the notebooks from `notebooks/`; `nbsphinx` just picked up the ones in `source/examples/`. When I added nbsphinx, I didn't have time to migrate all the notebooks from `notebooks` into the proper docs. That's likely why it's timing out.

**Comment by hussainsultan** [comment]
it actually just failed on `test_algo_families` Determinism[get2-func1-kwargs1] test....

**Comment by TomAugspurger** [comment]
I can take a look later tonight.

**Comment by agramfort** [comment]
Please take over. Won't have time to look soon

Alex

**Comment by moody-marlin** [comment]
For the test failure:

Replace [this line](https://github.com/dask/dask-glm/blob/master/dask_glm/utils.py#L26) with 
```
out = algo(Xn, y, *args, **kwargs).copy()
```
it's something weird with the distributed scheduler; I had a similar problem when I first wrote the `normalize` code (which yea, we should probably re-visit for multiple reasons at this point).

**Comment by agramfort** [comment]
pushed

**Comment by hussainsultan** [comment]
@TomAugspurger  do you think the timeouts are because of doc build? I also noticed that our tests take significant time to run. Happy to take on the work to get this passing again.

**Comment by agramfort** [comment]
feel free to merge / take over this PR

**Comment by TomAugspurger** [comment]
Sorry about the delay in getting back to this. I just pushed a commit that moves all the notebooks into the `docs/examples`. They'll be run automatically on readthedocs when the docs are built. Hopefully with Travis just doing testing, and RTD just doing docs, we won't hit any timeout issues.

I'll merge this once it's green, which should trigger the RTD build.

**Comment by hussainsultan** [comment]
Thank you @TomAugspurger  

Merging soon. if no comment.

**Comment by TomAugspurger** [comment]
Sorry got busy with other things. I'll followup if the doc build fails.
